### ISSUE TITLE
panic of the testground not logged to run.err or run.out

### ISSUE THREAD

**Issue by willscott** [ipfs/testground#674]
#### Version information:
From current master. using local docker builder/runner

#### Description:
Was debugging an issue in my autonat branch where some nodes would panic during the test.
This panic was emitted in the output of the testground daemon, but did not end up in either the `run.out` or `run.err` output of those nodes.
Since i had testground daemon  running in a screen that i wasn't looking at. it took overly long to notice that this was the problem.

I would expect the panic stacktrace to end up in the node's `run.err`.

**Comment by raulk** [comment]
It's weird that the panic didn't end up in `run.out`. You should've seen an error event like this one:

```
{"ts":1584020283315235000,"msg":"","group_id":"single","run_id":"2530cd4b96fc","event":{"type":"finish","outcome":"failed","error":"this is an intentional panic","stacktrace":"goroutine 1 [running]:\nruntime/debug.Stack(0x14bd575, 0x2, 0xc000185a70)\n\t/usr/local/Cellar/go/1.14/libexec/src/runtime/debug/stack.go:24 +0x9d\ngithub.com/ipfs/testground/sdk/runtime.(*logger).RecordCrash(0xc0002845e0, 0x14387e0, 0xc0002a2180)\n\t/private/var/folders/tv/ly66_c6121jg370fvnfcb_mh0000gn/T/placebo569081630/sdk/runtime/output.go:170 +0xa2\ngithub.com/ipfs/testground/sdk/runtime.Invoke.func4(0xc0002bc120)\n\t/private/var/folders/tv/ly66_c6121jg370fvnfcb_mh0000gn/T/placebo569081630/sdk/runtime/runner.go:90 +0x52\npanic(0x14387e0, 0xc0002a2180)\n\t/usr/local/Cellar/go/1.14/libexec/src/runtime/panic.go:967 +0x15d\nmain.run(0xc0002bc120, 0x0, 0x0)\n\t/private/var/folders/tv/ly66_c6121jg370fvnfcb_mh0000gn/T/placebo569081630/plan/main.go:42 +0x307\ngithub.com/ipfs/testground/sdk/runtime.Invoke(0x14dcb78)\n\t/private/var/folders/tv/ly66_c6121jg370fvnfcb_mh0000gn/T/placebo569081630/sdk/runtime/runner.go:98 +0x4e4\nmain.main()\n\t/private/var/folders/tv/ly66_c6121jg370fvnfcb_mh0000gn/T/placebo569081630/plan/main.go:13 +0x2d\n"}}
```

I've filed #678 to make sure that panics are printed on `run.err`. But even with the current version, the new placebo test case I added to test panics prints the panic on `run.out`.

**Comment by Robmat05** [comment]
I am labeling this as duplicate/discarded. Bug was addressed in #678
